src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -251,6 +269,10 @@ protected override void SerializeCore(XmlElement element, SaveContext context)
             XmlElement script = element.OwnerDocument.CreateElement("Script");
             script.InnerText = this.script;
             element.AppendChild(script);
+            XmlElement engine = element.OwnerDocument.CreateElement(nameof(Engine));
+            engine.InnerText = Enum.GetName(typeof(PythonEngineVersion), Engine);
+            element.AppendChild(engine);
```

Comment:
Seems these can still be easily converted to use Json in the future

Reply:
Agreed. The fact that there are things in Dynamo depending on XML serialization, when we have obsoleted it, is scary.

Reply:
I will file a task to do as spike on this, I believe we did one way back when we looked at JSON originally, maybe I can dig up some notes.